src/frontend/src/pages/CreateChangeRequestPage/CreateChangeRequestView.tsx

```diff
@@ -204,61 +221,76 @@ const CreateChangeRequestsView: React.FC<CreateChangeRequestViewProps> = ({
                 value={wbsDropdownOptions.find((element) => element.id === wbsNum) || null}
               />
             </Grid>
-            <Grid item xs={6}>
+            <Grid item xs={10}>
               <FormControl fullWidth>
                 <FormLabel>Type</FormLabel>
                 <Controller
                   name="type"
                   control={control}
                   rules={{ required: true }}
                   render={({ field: { onChange, value } }) => (
-                    <TextField select onChange={onChange} value={value}>
-                      {permittedTypes.map((t) => (
-                        <MenuItem key={t} value={t}>
-                          {t}
-                        </MenuItem>
-                      ))}
-                    </TextField>
+                    <RadioGroup onChange={onChange} value={value}>
+                      <Grid display="flex" flexDirection={{ xs: 'column', md: 'row' }}>
```

Comment:
you're using this like a Box component, so I would just use the Box component

Reply:
also this happens when the screen gets shrunk enough. I would use rowGap and columnGap props on the Box (or I think you can put them both in one) instead of margin stuff like you're doing for the horizontal space rn to fix that.
https://developer.mozilla.org/en-US/docs/Web/CSS/gap
<img width="708" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/fb0a1b09-b9a7-4fa5-8508-533dc724512e">